### ISSUE TITLE
The linter doesn't work when folder has parentheses in the name

### ISSUE THREAD

**Issue by alexander-lysenko-88** [DavidAnson/vscode-markdownlint#215]
Hi!

I've faced an issue with the linter, details are below.

Steps to reproduce:
1. Create a folder with parentheses in the name, e.g. `C:\test(1)\`
2. Create an md file to the folder, e.g. `test.md`
3. Open the file in VS Code and type some markdown with errors and save the file, e,g,
```
asdasdasd
 
 as
 d 
 dddddsd asd asd 
```
Actual result:
None validation error/warning is present in the VS Code PROBLEMS panel

Expected result:
Validation errors/warnings should be displayed.

4. Rename the folder e.g. `C:\test1\`
5. Open the file again

Actual result: validation warning are present now, and the behavior correspond the expected result.

**Comment by DavidAnson** [comment]
Duplicate of #207, though this issue has a better title.

**Comment by meling** [comment]
I have the same or similar issue. My scenario is the following:

Open VSCode workspace in:

```sh
> pwd
/Users/meling/work/distsys/dat520
> code .
```

However, the real path is this, because `work` is a symlink:

```sh
> realpath .
/Users/meling/Dropbox (Team UiS)/work/distsys/dat520
```

Unfortunately, I'm not able to rename my `Dropbox (Team UiS)` because it is beyond my control.
So my only workaround so far is to move my code folder out of Dropbox temporarily.
Hoping for a fix soon!

**Comment by DavidAnson** [comment]
It's coming. The library and CLI2 code are already updated.
